examples/src/models/chat/integration_googlevertexai-tools.ts

```diff
@@ -25,7 +25,7 @@ const geminiCalculatorTool: GeminiTool = {
 
 const model = new ChatVertexAI({
   temperature: 0.7,
-  model: "gemini-1.0-pro",
+  model: "gemini-1.5-flash-001",
```

Comment:
If you update these models you also need to re-run & update the outputs which are in the comment below, along with the LangSmith trace links in the mdx file.